projects/transformers/run.py

```diff
@@ -100,6 +102,11 @@ def main():
             training_args.output_dir, training_args.run_name
         )
 
+        # Initialize wandb now to include the logs that follow.
+        # For running and logging with multiple experiment, this is skipped for now.
+        if len(cmd_args.experiments):
```

Comment:
Isn't experiment a mandatory argument, why the if?

Reply:
That's a typo. It should be `if len(cmd_args.experiments) == 1`. There's work needed to log with wandb when running multiple experiments because one needs to start and stop the wandb run and then restart another. I looked into this, but I'm not sure it's possible, so I only covered the case I know work; just one experiment per run.